### PR TITLE
Pytest of grodecoder.py on all the file in data/exemples/

### PR DESCRIPTION
**Pull request by KarinDuong**
Pytest of grodecoder.py on all the file in data/exemples/, by comparing the JSON output (for the key 'inventory') that we get with a reference output (in tests/data/)

### CODE REVIEW

**Comment by pierrepo** [comment]
@KarinDuong could you remove `step7_1.gro` and `step_1.json`?

File: tests/test_grodecoder.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+)
+def test_with_param(filepath_test, JSON_filepath_data):
+    JSON_filepath = gd.main(filepath_test)
+    with open(JSON_filepath, 'r') as f:
+        JSON_content_test = json.load(f)
+
+    with open(JSON_filepath_data, 'r') as f:
+        JSON_content_data = json.load(f)
+
+    assert JSON_content_test["inventory"] == JSON_content_data["inventory"]
```
[the remark this answers is not in the document]
Comment (KarinDuong) [reply tests/test_grodecoder.py:37]: Done ! :white_check_mark:

**Comment by pierrepo** [comment]
> @KarinDuong could you remove `step7_1.gro` and `step_1.json`?

Also `step7_1.gro`

**Comment by pierrepo** [comment]
@KarinDuong `test_grodecoder.py` should stay in the `tests` directory

**Review by pierrepo — CHANGES_REQUESTED** [review]

**Comment by pierrepo** [comment]
@KarinDuong could you please blackify your code?

**Review by pierrepo — APPROVED** [review]

**Comment by pierrepo** [comment]
@KarinDuong code looks good. Could we merge?

**Pull request merged**